campaign.html

```diff
@@ -25,6 +25,16 @@
 </div>
 
 <div class="margin-top">
+  <p class="stats">
+    <strong>The fundraising campaign for RGSoC 2017 is now open!</strong>
+  </p>
+
+  <p>
+    All the amount we receive from this moment on will be used on next year's edition.
```

Comment:
Can you change "All the amount" to "All the money"? And "will be used on" -> "will be used for".